mainwindow.cpp

```diff
@@ -278,19 +289,13 @@ MainWindow::~MainWindow()
 void MainWindow::on_pushButton_WordlistFileBrowse_clicked()
 {
     // We pop a dialog to choose a file to open.
-    // TODO: *.lst for file extension?
-    // TODO: Copy-pasting is evil! (open password file)
-    // TODO: What happens when John writes something while dialog
-    //       opened?
-    // TODO: Move dialog creation and setting up into window constructor.
     QFileDialog dialog;
     dialog.setFileMode(QFileDialog::ExistingFile);
-    // TODO: Dialog could allow user to select multiple files. May it
-    //       be good to support this ability? To concatenate selected file?
```

Comment:
I guess we can make it so if multiple files are selected in the File dialog we just append them in that order to the JohnProcess arguments list and that's it. Nothing more.